### DISCUSSION TITLE
Whitelist function from wiki

### DISCUSSION THREAD

**Discussion by chrisanthropic** [CyberShadow/aconfmgr#133] [Q&A]
Are folks still using the whitelist function in the wiki?

I've created a `00-helpers.sh` file with the function and have tried adding the example snippet below to both the `00-helpers.sh` and `10-ignores.sh` individually, but neither shows any changes when running `aconfmgr save`.

```
white_list=(
    '.gitconfig'
    '.config/*'
    '.kde4/share/config/*'
)
IgnorePathsExcept ~ "${white_list[@]}"
```

**Comment by CyberShadow** [comment]
For the record, here is the discussion which led to the page being created:
https://github.com/CyberShadow/aconfmgr/issues/12#issuecomment-505747159
